### DISCUSSION TITLE
[Vote ended on 2022-11-08] Collection Removal Policy: add a paragraph about Collection Requirements violations

### DISCUSSION THREAD

**Discussion by Andersson007** [ansible-community/community-topics#158] [Votes done]
Vote on merge the [PR](https://github.com/ansible-collections/overview/pull/217), which adds a paragraph to exclusion policy explicitly covering cases that don't belong to unmaintained/broken collections.

Discussion: https://github.com/ansible-community/community-topics/issues/150
PR: https://github.com/ansible-collections/overview/pull/217

Please vote with `+1/-1`

**Comment by Andersson007** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by maxamillion** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by gundalow** [comment]
+1 to merge https://github.com/ansible-collections/overview/pull/217 I appreciate everybody that's worked on this.
